rpcserver.go

```diff
@@ -3499,19 +3499,17 @@ func handleGetCoinSupply(s *rpcServer, cmd interface{}, closeChan <-chan struct{
 		return s.coinSupplyTotal, nil
 	}
 
-	for i := 0; int64(i) < tipHeight+1; i++ {
-		if i == 0 {
-			continue
-		}
+	for i := 0; i < int(tipHeight/params.ReductionInterval); i++ {
```

Comment:
If I understand correctly the vote bits refer to transactions in the previous block, therefore I found it useful to reverse the order of the blocks in the for loop below. Since the order is reversed, base is now set to the latest subsidy initially.